riotfile.py

```diff
@@ -2422,7 +2422,7 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
         Venv(
             name="dbapi_async",
             command="pytest {cmdargs} tests/contrib/dbapi_async",
-            pys=select_pys(min_version="3.5"),
+            pys=select_pys(min_version="3.6"),
```

Comment:
What's the issue with python3.5?

Reply:
One of the classes we use throughout the tests, AsyncMock, isn't available. Also, not really a need to test for 3.5 considering there is no async support anyways.